tests/test_taipy.py

```diff
@@ -435,6 +434,20 @@ def test_cycle_exists(self):
             tp.exists(cycle_id)
             mck.assert_called_once_with(cycle_id)
 
+    def test_create_global_data_node(self):
+        dn_cfg = DataNodeConfig("id", "pickle", Scope.GLOBAL)
+        with mock.patch("src.taipy.core.data._data_manager._DataManager._create_and_set") as mck:
+            dn = tp.create_global_data_node(dn_cfg)
+            mck.assert_called_once_with(dn_cfg, None, None)
+
+        dn = tp.create_global_data_node(dn_cfg)
+        assert dn.scope == Scope.GLOBAL
+        assert dn.config_id == dn_cfg.id
+
+        dn_cfg.scope = Scope.SCENARIO
+        with pytest.raises(DataNodeConfigIsNotGlobal):
+            tp.create_global_data_node(dn_cfg)
+
```

Comment:
What happens when we create two data nodes from the same GLOBAL data node config ?